app/src/main/res/values/strings.xml

```diff
@@ -163,21 +163,21 @@
     <string name="privacy_policy_popup_text">Privacy policy notice (required by some app stores):\n\nNO DATA IS COLLECTED AT ALL, which anyone can confirm since our app is libre software.</string>
     <string name="accept">Accept</string>
     <string name="importCatima">Import from Catima</string>
-    <string name="importCatimaMessage">Find a file likely named <i>Catima.csv</i> to import.\nOr create it from the Import/Export menu of another Catima app by pressing "Export" there first.</string>
+    <string name="importCatimaMessage">Find a file likely named <i>Catima.csv or .zip</i> to import.\nOr create it from the "Import/Export" menu of another Catima app by pressing "Export" there first.</string>
```

Comment:
Not sure if the "or" should be in or not. Can't seem to be engaged with the opportunity of either.

Reply:
```suggestion
    <string name="importCatimaMessage">Find a file named <i>Catima.csv or .zip</i> to import.\nOr create it from the "Import/Export" menu of another Catima app by pressing "Export" there first.</string>
```
Or are there yet more options?

Reply:
Maybe we should only mention catima.zip, because catima.csv is legacy. Just keeps it more clean given 99.9% of cases will use the .zip